cpp/src/arrow/filesystem/CMakeLists.txt

```diff
@@ -28,6 +28,23 @@ add_arrow_test(filesystem-test
                EXTRA_LABELS
                filesystem)
 
+if(ARROW_BUILD_BENCHMARKS AND ARROW_PARQUET)
+    add_arrow_benchmark(localfs_benchmark
+                        PREFIX
+                        "arrow-filesystem"
+                        SOURCES
+                        localfs_benchmark.cc
+                        STATIC_LINK_LIBS
+                        ${ARROW_BENCHMARK_LINK_LIBS}
+                        Boost::filesystem
+                        Boost::system)
```

Comment:
Looks like the Boost dependency isn't needed?

Reply:
You are right, this should not be needed?